### ISSUE TITLE
Feedback system for event objects

### ISSUE THREAD

**Issue by ahf** [bornhack/bornhack-website#388]
It would be nice to have a feedback system for events on the website. This would make it easier for the content team to make decisions about which speakers we can accept and which ones we reject.

I think the following model would be nice:

Participants submit feedback on the website, whilst being logged in, with the following information:

- Did the event live up to your expectations? Yes/No (boolean)
- Would you attend another event with the given speaker? Yes/No (boolean)
- Rating/Score (An integer from 0 to 5)
- Comment (Free text)

The content team should be able to see who the feedback author is and approve the feedback before it is visible to the event owner. The event owner should not see the author of the feedback and would see all comments as being anonymous. The approval part is to prevent abuse and/or prevent useless comments. Feedback should by default be available for *all* event objects, but should be possible to disable for things such as "Lunch break" and other things.

**Comment by tykling** [comment]
do we want users to be able to provide feedback for Event or EventInstance objects? In other words, should it be possible to provide feedback twice for the same Event, for example if attending the same workshop twice (different EventInstances)?

**Comment by tykling** [comment]
we want to provide feedback for Event objects, not EventInstance objects (clarified on IRC)

**Comment by ahf** [comment]
It seems to look good. I think the only changes that would be nice to have is:

- Instead of having 0 to 5 as scoring, have it as 1 to 5.
- Instead of using radio buttons use a slider for the scoring.
- Instead of radio buttons for yes/no questions use something like a checkbox.

**Comment by tykling** [comment]
reopening until we fix these last things

**Comment by zarya** [comment]
Will fix this after #1667
